clinica/pipelines/t1_linear/t1_linear_pipeline.py

```diff
@@ -0,0 +1,258 @@
+# -*- coding: utf-8 -*-
+"""T1 Linear - Clinica Pipeline.
+This file has been generated automatically by the `clinica generate template`
+command line tool. See here for more details: https://gitlab.icm-institute.org/aramislab/clinica/wikis/docs/InteractingWithClinica.
+"""
+import clinica.pipelines.engine as cpe
+
+
+class T1Linear(cpe.Pipeline):
+    """T1 Linear SHORT DESCRIPTION.
+    This preprocessing pipeline includes globally three steps:
+    1) N4 bias correction (performed with ANTS).
+    2) Linear registration to MNI (MNI icbm152 nlinear sym template)
+      (performed with ANTS) - RegistrationSynQuick.
+    3) Cropping the background (in order to save computational power).  4)
+    Histogram-based intensity normalization. This is a custom function
+    performed by the binary ImageMath included with ANTS.
+
+    Warnings:
+        - A WARNING.
+
+    Args:
+        bids_directory (str): Folder with BIDS structure.
+        caps_directory (str): Folder where CAPS structure will be stored.
+        ref_template (str): reference template used for image registration.
+        working_directory (str): Folder containing a temporary space to save.
+        intermediate results.
+        tsv: The Subjects-Sessions list file (in .tsv format).
+
+    Returns:
+        A clinica pipeline object containing the T1 Linear pipeline.
+
+    Raises:
+
+    Example:
+        >>> from t1_linear import T1Linear
+        >>> pipeline = T1Linear('~/MYDATASET_BIDS', '~/MYDATASET_CAPS')
+        >>> pipeline.parameters = {
+        >>>     # ...
+        >>> }
+        >>> pipeline.base_dir = '/tmp/'
+        >>> pipeline.run()
+    """
```

Comment:
Proposition:
```python
class T1Linear(cpe.Pipeline):
    """T1Linear - Affine registration of T1w images to standard space.

    This pipeline performs globally three steps:
    1) Bias correction with N4 algorithm from ANTs.
    2) Linear registration to MNI152NLin2009cSym template
    with RegistrationSynQuick from ANTs.
    3) Crop of the the background (in order to save computational power)

    Returns:
        A clinica pipeline object containing the T1Linear pipeline.
    """
```
Currently, pipeline classes do not contain pipeline args section nor example section. This can be rediscussed when a Sphinx or similar tool will be added to Clinica.

Reply:
>> Currently, pipeline classes do not contain pipeline args section nor example section. This can be rediscussed when a Sphinx or similar tool will be added to Clinica.

That's a pity! We should start thinking about fully documented functions from now!
I accept you proposition in order to get forward with the PR.